{
	"identifier": "USN-4888-2",
	"description": "",
	"type": "none",
	"severity": "none",
	"state": "invalid",
	"datetime": "0000-00-00T00:00:00Z",
	"is_edited": false,
	"hardware": [],
	"software": [
		{
			"name": "ldb-tools",
			"version": "\u003c 1:1.1.24-0ubuntu0.14.04.2+esm1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-trusty"
		},
		{
			"name": "libldb-dev",
			"version": "\u003c 1:1.1.24-0ubuntu0.14.04.2+esm1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-trusty"
		},
		{
			"name": "python-ldb-dev",
			"version": "\u003c 1:1.1.24-0ubuntu0.14.04.2+esm1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-trusty"
		},
		{
			"name": "python-ldb",
			"version": "\u003c 1:1.1.24-0ubuntu0.14.04.2+esm1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-trusty"
		},
		{
			"name": "libldb1",
			"version": "\u003c 1:1.1.24-0ubuntu0.14.04.2+esm1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-trusty"
		}
	],
	"products": [],
	"references": [
		"https://ubuntu.com/security/notices/USN-4888-2"
	],
	"weaknesses": []
}